smoke_tests/happypath/resume.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Resume to continue existing application
+
+  Background:
+    Given I show my environment
+    Then I visit "/"
+    And I should see "What do you want to do?"
+    When I click the "Appeal against a tax decision" link
+    Then I should see "Appeal against a tax decision"
+    And I click the "Continue" link
+    Then I should see "What is your appeal about?"
+    Given I choose "Income Tax"
+    Then I should see "Did you appeal the original decision to HMRC?"
+
+    Given I choose "Yes"
+    And I choose "I have a review conclusion letter"
+    Then I should see "What is your dispute about?"
+
+    Given I click the "Save and come back later" link
+    And I fill in my email address
+    And I fill in "Choose password" with "ABCD1234"
+    When I click the "Save" button
+    Then I should see "Your case has been saved"
+    Given I click the "Sign in" link
+    And I fill in my email address
+    And I fill in "Enter password" with "ABCD1234"
+    When I click the "Sign in" button
+    Then I should see "Your saved cases"
+
```

Comment:
You can also put all this in a step, like this:


```
Given(/^I create a saved case$/) do
  step 'I show my environment'
  step 'I visit "/"'
  step 'I click the "Appeal against a tax decision" link'
  step 'I click the "Continue" link'
  step 'I choose "Income Tax"'
  step 'I choose "Yes"'
  step 'I choose "I have a review conclusion letter"'

so on and so on until you complete all the steps to create a saved case...
end
```
Then the background could just read:
```
Background: Create a saved case
  Given I create a saved case
```